source/components/actionbar.rst

```diff
@@ -0,0 +1,14 @@
+==================
+Action Bar
+==================
+
+Send messages to the action bar, located above the health bar of a player.
+
+.. code-block:: java
+
+    public void sendMyActionBar(final Audience server) {
+        final Component actionBar = Component.text("You smell funny");
+        server.sendActionBar(actionBar);
+    }
+
+It is not possible to control fade-in/out. For similar functionality, see :ref:`title`.
```

Comment:
Is this really similar? if so, go over the key differences

Reply:
Location is the other key difference which is mentioned in the first sentence. That and fade-in/out are the only differences I'm aware of. I imagine this is why they are both arguments under the same vanilla `/title` command.